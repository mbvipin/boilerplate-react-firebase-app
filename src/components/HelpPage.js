import React from 'react';


const HelpPage=()=>(

    <div>
  <p> This is Help page</p>

    </div>

);

export default HelpPage;