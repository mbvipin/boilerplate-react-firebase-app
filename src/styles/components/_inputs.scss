.text-input
{
    border: 1 px solid hsl(200, 3%, 80%);
    height: 50px;
    font-size: $font-size-large;
    font-weight:300;
    padding: $s-size;
}

.select
{
    @extend .text-input;
}

.textarea
{
    @extend .text-input;
    height: 10rem;
}