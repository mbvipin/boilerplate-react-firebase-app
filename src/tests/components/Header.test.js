import React from 'react';
import { shallow } from 'enzyme';
import {Header} from '../../components/Header';

test('should render Header', () => {

    const wrapper = shallow(<Header startLogOut={()=>{}} />);

    expect(wrapper).toMatchSnapshot();

});

test('should handle onClick for Log out ', () => {

    const startLogOut=jest.fn();
    
    const wrapper = shallow(<Header startLogOut={startLogOut} />);

    wrapper.find('button').simulate('click');

    expect(startLogOut).toHaveBeenCalled();
})

