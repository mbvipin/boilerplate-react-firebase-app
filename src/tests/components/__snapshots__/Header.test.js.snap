// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header 1`] = `
<header
  className="header"
>
  <div
    className="content-container"
  >
    <div
      className="header__content"
    >
      <Link
        className="header__title"
        replace={false}
        to="/dashboard"
      >
        <h1>
          Boilerplate
        </h1>
      </Link>
      <button
        className="button button--link"
        onClick={[Function]}
      >
        Log Out
      </button>
    </div>
  </div>
</header>
`;
